### PR TITLE
fix integration test screenshots [SATURN-1212]

### PR DESCRIPTION
**Pull request by panentheos**
* Adds necessary `png` extension to filename.
* Makes screenshot capture opt-in via the environment variable.
* Explicitly creates the directory on Circle.

Tested locally and on Circle by temporarily pushing a breaking change.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -159,6 +159,7 @@ jobs:
             JEST_JUNIT_OUTPUT_DIR: "/tmp/test-results/integration-test-results.xml"
             SCREENSHOT_DIR: "/tmp/failure-screenshots"
           command: |
+            mkdir -p /tmp/failure-screenshots
```
Comment (breilly2) [inline .circleci/config.yml]: Assuming we have access to the env above, this could be `mkdir -p ${SCREENSHOT_DIR}`.

File: integration-tests/utils/integration-config.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const {
   TERRA_TOKEN: bearerToken,
   BILLING_PROJECT: billingProject = 'general-dev-billing-account',
   WORKFLOW_NAME: workflowName = 'haplotypecaller-gvcf-gatk4',
-  SCREENSHOT_DIR: screenshotDir = 'failure-screenshots'
```
Comment (breilly2) [inline integration-tests/utils/integration-config.js:6]: TOL: I'm ever so slightly in favor of capturing screenshots by default. If we assume that the tests are pretty stable, then we'll most often see failures in edge cases that we might not be able to readily reproduce after opting-in.

**Review by breilly2 — APPROVED** [review]
👍 In this case, I did not test myself. I trust the testing that you explicitly said you did.

File: integration-tests/utils/integration-config.js (hunk 2 above)
Comment (panentheos) [reply integration-tests/utils/integration-config.js:6]: Let's revisit this in a followup PR. It's a fair point, I just want to make sure we get the management of the files right.

**Review by breilly2 — APPROVED** [review]
:+1: Thanks for fixing this!
